src/com/cybozu/labs/langdetect/util/LangProfile.java

```diff
@@ -47,6 +47,25 @@ public void add(String gram) {
     }
 
     /**
+     * Merge two language profiles together
+     * @param other other LangPorfile
+     */
+    public void merge(LangProfile other){
+        if(! this.name.equals(other.name)) return; // Illegal
+        for(int i =0; i < n_words.length; i++){
+            n_words[i] += other.n_words[i];
+        }
+        Set<String> keys = other.freq.keySet();
+        for(Iterator<String> i = keys.iterator(); i.hasNext();){
+            String key = i.next();

```

Comment:
I did. The rest of the project doesn't use it (or if it does, does so inconsistently). I'd prefer to have the code consistent throughout rather than sprinkling in minor stylistic differences.